docs/asyncapi.yaml

```diff
@@ -291,6 +348,82 @@ components:
       type: number
       description: Key field for calculation Data 
       example: 1234567
+    ApplicationId:
+      type: number
+      description: Key field for calculation Data 
+      example: 1234567
+    PaymentBand1:
+      type: number
+      description: first payment band bracket amount 
+      example: Up to £30000
+    PaymentBand2:
+      type: number
+      description: second payment band bracket amount 
+      example: £30,000.01 to £50,000
+    PaymentBand3:
+      type: number
+      description: third payment band bracket amount 
+      example: £50,000.01 to £150,000
+    PaymentBand4:
+      type: number
+      description: fourth payment band bracket amount 
+      example: Above £150,000
+    PercentageReduction1:
+      type: number
+      description: First percentage bracket for progressive reductions 
+      example: 50
+    PercentageReduction2:
+      type: number
+      description: Second percentage bracket for progressive reductions 
+      example: 55
+    PercentageReduction3:
+      type: number
+      description: Third percentage bracket for progressive reductions 
+      example: 60
+    PercentageReduction4:
+      type: number
+      description: Fourth percentage bracket for progressive reductions 
+      example: 70
+    ProgressiveReductions1:
+      type: number
+      description: Value against band1 for progressive reductions 
+      example: 11000
+      format: decimal
```

Comment:
not really an issue but should probably have these in the same order as the below ones, ie

- type
- format
- description
- example